presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -2153,6 +2153,20 @@ public void revokeTablePrivileges(ConnectorSession session, SchemaTableName sche
         return result.build();
     }
 
+    @Override
+    public void commitPartitions(ConnectorSession session, ConnectorOutputTableHandle tableHandle, Collection<Slice> fragments)
```

Comment:
nit: `commitPartition` since it's one partition at a time commit.

Reply:
We use the word "partition" here in terms of the Presto "partition", that in Hive maps to "bucket". In Hive 1 bucket = 1 "partition"  = 1 file = 1 fragment (partition update).

Reply:
`commitFiles` could happen because for each presto partition, we might need to write to multiple hive partitions, and thus multiple directories. In each directory there would be one file. One fragment in Hive (`PartitionUpdate`) is one directory (as you could see, there is one path and a list of file names).

Reply:
I see. Thanks for the explanation.